lib/crypto/eosdart/src/models/account.g.dart

```diff
@@ -118,16 +82,13 @@ Map<String, dynamic> _$PermissionToJson(Permission instance) =>
 RequiredAuth _$RequiredAuthFromJson(Map<String, dynamic> json) {
   return RequiredAuth()
     ..threshold = json['threshold'] as int?
-    ..keys = (json['keys'] as List?)
-        ?.map((e) =>
-            e == null ? null : AuthKey.fromJson(e as Map<String, dynamic>))
-        .toList()
-    ..accounts = (json['accounts'] as List?) as List<Object>?
-    ..waits = (json['waits'] as List?) as List<Object>?;
+    ..keys =
+        (json['keys'] as List?)?.map((e) => e == null ? null : AuthKey.fromJson(e as Map<String, dynamic>)).toList()
```

Comment:
we should be consistent and use List.from everywhere there's an "as List?"

Also I think as List? is a mistake by the library since these calls never return null, they always return a list. Just an empty list if that item doesn't exist.